exporter/awsemfexporter/metric_translator.go

```diff
@@ -169,6 +205,25 @@ func (mt metricTranslator) translateOTelToGroupedMetric(rm pmetric.ResourceMetri
 	return nil
 }
 
+func fetchEntityFields(resourceAttributes pcommon.Map) (cloudwatchlogs.Entity, pcommon.Map) {
+	// the original resourceAttributes map is immutable, so we need to create a mutable copy
+	// to remove the entity fields from the attributes
+	mutableResourceAttributes := pcommon.NewMap()
+	resourceAttributes.CopyTo(mutableResourceAttributes)
+	serviceKeyAttr := map[string]*string{
+		entityType: aws.String(service),
```

Comment:
Same here are we referring to type in https://w.amazon.com/bin/view/AWS/AIO/Specs/O2F#H2.1.1TypeService? why the name is entityType?

Reply:
entityType is the variable name for Type https://github.com/amazon-contributing/opentelemetry-collector-contrib/pull/233/files#diff-af3bf0e3a5a889da31842d592ffeecc4ef5d3e5b058af7ca500ed44e321bca0cR44